packages/fxa-auth-server/docs/swagger/shared/descriptions.ts

```diff
@@ -38,7 +51,13 @@ const DESCRIPTIONS = {
     'The [PKCE](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/oauth/pkce.md) code verifier. Required for public clients, forbidden otherwise.',
   command:
     "The id of the command to be invoked, as found in the device's availableCommands set.",
+  createdAt: 'This is the date the subscription was created.',
   createdTime: 'Integer time of token creation.',
+  currency: 'The three-letter ISO currency code, in lowercase.',
+  currencyCode: 'The three-letter ISO currency code, in uppercase.',
```

Comment:
Seems a little redundant... I know you're just reflecting back what we're using in our routes.

Reply:
As discussed, `currencyCode` is case-sensitive, as it includes uppercase in its validation.